### ISSUE TITLE
[BUG] HttpService from @nestjs/common package is deprecated

### ISSUE THREAD

**Issue by cpoyatos1** [OpenAPITools/openapi-generator#13376]
### Description

After generating the code I get this deprecation warning in nestjs v7.x.
```
"HttpService" (from the "@nestjs/common" package) is deprecated and will be removed in the next major release. Please, use the "@nestjs/axios" package instead.
```
In nestjs v8.x. "HttpService" has been removed, so this doesn't work at all anymore.

#### openapi-generator version

`@openapitools/openapi-generator-cli@2.5.2`

### Generation Details

```
openapi-generator-cli generate -g typescript-nestjs -i openapi.yaml
```

**Comment by jasperx21** [comment]
this fixed in 6.1.0

**Comment by cpoyatos1** [comment]
works
